model/scope/exporter/exporter.yaml

```diff
@@ -6,10 +6,12 @@ groups:
     attributes:
       - id: name
         type: string
+        stability: experimental
```

Comment:
I think these should be stable, no? https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/common/mapping-to-non-otlp.md

Reply:
I'm glad we're doing this explicitly :) 

Let me send separate PRs to add stability there